src/python/pants/ivy/ivy_subsystem.py

```diff
@@ -39,7 +39,14 @@ def register_options(cls, register):
     register('--ivy-profile', advanced=True, default=cls._DEFAULT_VERSION,
              help='The version of ivy to fetch.')
     register('--cache-dir', advanced=True, default=os.path.expanduser('~/.ivy2/pants'),
-             help='Directory to store artifacts retrieved by Ivy.')
+             help='The default directory used for both the Ivy resolution and repository caches.'
+                  'If you want to isolate the resolution cache from the repository cache, we '
```

Comment:
Nit: Is it worth requiring that both `resolution` and `repository` is set if either are?